tests/marqo_test.py

```diff
@@ -1,8 +1,22 @@
-"""Please have a running Marqo instance to test against!
-
-
-Pass its settings to local_marqo_settings.
 """
+Before running the tests locally, ensure that you have a running Marqo instance to test against!
+Pass its settings to the local_marqo_settings.
+
+To run Cloud V2 tests, execute `tox -e cloud_tests`.
+When running Cloud V2 tests, make sure to set the following environment variables:
+- MARQO_CLOUD_URL: The URL that the config class uses to recognize the cloud environment.
+- MARQO_API_KEY: The API key used for authentication.
+- MARQO_URL: The URL of the Marqo cloud instance.
+
+When `cluster.is_marqo_cloud` is set to True, the tests will have different setUp and tearDown procedures:
+- Indices will not be deleted after each test.
+- However, documents will be deleted after each test.
+
+The function `create_cloud_index` will be triggered whenever an index needs to be created for the cloud tests.
+It checks the status of the index and creates a new index if needed. It also adds a suffix for unique test execution
+and a hash generated from index settings to ensure each index is unique.
```

Comment:
Let's make this sentence more accurate:

~~a hash generated from index settings to ensure each index is unique.~~
->
...a hash generated from index settings ensure that only 1 index is created for a given index settings_dict or index kwargs. It also ensures that any tests that require an index with specific settings get routed to the appropriate index.